blurb/blurb.py

```diff
@@ -745,19 +745,18 @@ def test(*args):
 
 
 def find_editor():
-    for var in 'GIT_EDITOR', 'EDITOR':
-        editor = os.environ.get(var)
-        if editor is not None:
-            return editor
```

Comment:
Perhaps we should keep this bit as a way to customize blurb usage apart from git.  Particularly since we still have an error message below that says you should set EDITOR (which no longer directly does anything here).